Documentation/reports.md

```diff
@@ -241,7 +241,33 @@ spec:
   reportingEnd: "2019-07-31T00:00:00Z"
 ```
 
+### expiration 
+
+Add the expiration field to set a retention period on a scheduled metering Report. You can avoid manually removing the Report by setting the expiration duration value. The retention period is equal to the Report creationDate plus the `expiration` duration. The Report is removed from the cluster at the end of the retention period if no other Reports or ReportQueries depend on the expiring Report. Deleting the Report from the cluster can take several minutes.
```

Comment:
We may want to specify that it's `metadata.creationDate` rather than simple `creationDate` without more context.